### PR TITLE
Tiny string fix

### PR DESCRIPTION
**Pull request by Howard-C**

### CODE REVIEW

File: mscore/editstaff.ui [hunk 1]
```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>760</width>
-    <height>600</height>
+    <width>895</width>
+    <height>657</height>
```
Comment (Howard-C) [inline mscore/editstaff.ui]: Automatic Qt Designer change. The updated values are the minimum on my computer.

Comment (Jojo-Schmitz) [reply mscore/editstaff.ui]: you're not forced to commit this part

Comment (Howard-C) [reply mscore/editstaff.ui]: Just trying to fix as soon as possible.

Comment (Spire42) [reply mscore/editstaff.ui]: Aside from this being unrelated to strings, I don't know if it actually fixes anything. Is it possible that the original 760×600 was chosen for a good reason? Maybe it works better on other OSes or at other DPI settings (or something else)?

Comment (Howard-C) [reply mscore/editstaff.ui]: ...

**Pull request merged**